.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: csharp
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+    - zlib1g
+
+matrix:
+  include:
+    # Ubuntu 14.04
+    - os: linux 
+      dist: trusty
+      dotnet: "1.0.0-preview2-003131"
+      env: DOTNET_SDK_URL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-ubuntu-x64.latest.tar.gz
```

Comment:
Where can I find the links to the specific versions? I wanted to pin it to a specific version, but all I could find was the link here - https://github.com/dotnet/cli#installers-and-binaries